src/test/java/teammates/test/cases/ui/browsertests/AdminEmailPageUiTest.java

```diff
@@ -66,22 +79,169 @@ private void testCompose() {
+                                            + "some more text. It cannot be longer than 45 characters. "
+                                            + "It cannot be empty and it cannot have spaces.");
+    }
+    
+    private boolean hasStatusMessageNoSubject() {
+        return emailPage.getStatus().equals("\"\" is not acceptable to TEAMMATES as email subject because "
+                                          + "it is empty. The value of email subject should be no longer "
+                                          + "than 200 characters. It should not be empty.");
+    }
+    
+    private boolean hasNoErrorMessage() {
+        return !(emailPage.isElementPresent(By.className("alert-danger")));
+    }
+    
+    private boolean hasStatusMessageSaveSuccess() {
+        return emailPage.getStatus().equals("Email draft has been saved");
+    }
+    
+    /**
+     * This method only checks if the email sent data table is displayed correctly
+     * i.e, table headers are correct
+     * It does not test for the table content
+     */
+    private boolean isEmailSentDataDisplayCorrect() {
+        if (emailPage.isElementPresent(By.className("table"))) {
+            return isEmailSentTableHeaderCorrect();
+        } else {     
+            return false;
+        }
+    }
+
+    private boolean isEmailSentTableHeaderCorrect() {
+        int numColumns = emailPage.getNumberOfColumnsFromDataTable(0); // 1 table
+        
+        if (numColumns != ADMIN_EMAIL_SENT_TABLE_NUM_COLUMNS) {
+            return false;
+        }
+        
+        List<String> expectedSessionTableHeaders = Arrays.asList("Action",
+                                                                 "Address Receiver",
+                                                                 "Group Receiver",
+                                                                 "Subject",
+                                                                 "Date");
+        List<String> actualSessionTableHeaders = new ArrayList<String>();
+        
+        for (int i = 0 ; i < numColumns ; i++) {
+            actualSessionTableHeaders.add(emailPage.getHeaderValueFromDataTable(0, 0, i));
+        }
+        
+        return actualSessionTableHeaders.equals(expectedSessionTableHeaders);
+    }
+    
+    /**
+     * This method only checks if the email draft data table is displayed correctly
+     * i.e, table headers are correct
+     * It does not test for the table content
+     */
+    private boolean isEmailDraftDataDisplayCorrect() {
+        if (emailPage.isElementPresent(By.className("table"))) {
+            return isEmailDraftTableHeaderCorrect();
+        } else {     
+            return false;
+        }
+    }
+
+    private boolean isEmailDraftTableHeaderCorrect() {
+        int numColumns = emailPage.getNumberOfColumnsFromDataTable(0); // 1 table
+        
+        if (numColumns != ADMIN_EMAIL_DRAFT_TABLE_NUM_COLUMNS) {
```

Comment:
Is this constant the only difference from `isEmailSentTableHeaderCorrect()`?
Why not pass it in as a parameter?